adapters/stroeerCore/stroeercore_test.go

```diff
@@ -0,0 +1,19 @@
+package stroeerCore
+
+import (
+	"github.com/prebid/prebid-server/adapters/adapterstest"
+	"github.com/prebid/prebid-server/config"
+	"github.com/prebid/prebid-server/openrtb_ext"
+	"testing"
+)
+
+func TestJsonSamples(t *testing.T) {
+	bidder, buildErr := Builder(openrtb_ext.BidderStroeerCore, config.Adapter{
+		Endpoint: "http://localhost:8361/s2sdsh"})
```

Comment:
Consider removing the port from the tests. IMHO it makes it look like that has important, when it should not.